### DISCUSSION TITLE
TCR target prediction

### DISCUSSION THREAD

**Discussion by Ahmedalaraby20** [satijalab/seurat#5911] [Q&A]
Hey guys, I hope you all are doing well, I have a list of full mouse TCR sequences that I got from my vdj scRNA seq experiment, I am just wondering if there is a tool that might help me to predict what these TCR might recognize?

**Comment by samuel-marsh** [comment]
Hi,

Not member of dev team but hopefully can be helpful.  This is not really a function of Seurat and therefore overall you will probably have better luck with getting good feedback on bioinformatics focused forum rather than software issues repository.

That said there are a number of papers which have started to address this issue in past couple years (though it is still very much in the early days, especially using single cell data) so I suggest google scholar/pubmed search as your best bet to familiarize yourself with techniques and packages that might suit your needs.  As caveat though much of the research on this has been from the human TCR side of things based on curated libraries of known TCR antigen bindings.  There are likely significant species differences at work and therefore I would be sure to focus on techniques that may have done this with mouse data as opposed to human patients.

Best,
Sam
